tap_google_analytics/client.py

```diff
@@ -13,12 +13,29 @@
 
 LOGGER = singer.get_logger()
 
+def is_retryable_403(e):
+    response = e.response
+    if not _is_json(response):
+        return False
+
+    retryable_errors = ["userRateLimitExceeded", "rateLimitExceeded", "quotaExceeded"]
+    error_reasons = [error.get('reason') for error in response.json().get('errors',[])]
+    for retryable_error in retryable_errors:
```

Comment:
You might be able to reduce this looping if you import something to check the two sets for an intersection?

Reply:
Oh, good call.